nlp_primitives/mean_characters_per_sentence.py

```diff
@@ -31,12 +31,9 @@ def get_function(self):
         def _mean_characters_per_sentence(text):
             if not isinstance(text, str):
                 return np.nan
-            if len(text) == 0:
-                return 0
-            sentences = sent_tokenize(text)
-            total = 0.0
-            for s in sentences:
-                total += len(s)
+            if not len(sentences := sent_tokenize(text)):
```

Comment:
Might just be me, but I find this construct to take more cognitive energy to understand than:
```
sentences = sent_tokenize(text)
if not len(sentences):
    ...
```

Also it seems like it makes it a little more difficult to find where sentences was created since we can end up using sentences outside of the `if` block in which the assignment was created.

Not a blocker for me one way or the other, but I would tend to prefer an approach without the walrus operator here.